src/tools/materialphysicalproperties/slags.py

```diff
@@ -127,12 +125,129 @@ def calculate(self, **state):
         x = amount_fractions(y)
         return super().calculate(T=T, x=x)
 
+class ModifiedUrbainViscosityTx(Model):
+    """
+    A model that describes the variation in the dynamic viscosity of liquid
+    slag as a function of temperature and composition expressed in mole
+    fraction. This "Modified" Urbain Model is more appropriate for Basic
+    Oxygen Steelmaking Slags than the standard, generic UrbainViscosity model.
+    """
+
+    def __init__(self):
+        state_schema = {'T': {'required': True, 'type': 'float', 'min': 0.0},
+                        'x': {'required': True, 'type': 'dict'}}
+        super().__init__('Slag', 'Dynamic Viscosity', 'mu', '\\mu', 'Pa.s',
+                         state_schema, None, None)
+
+    def calculate(self, **state):
+        """
+        Calculate dynamic viscosity at the specified temperature and
+        composition:
+
+        :param T: [K] temperature
+        :param x: [mole fraction] composition dictionary , e.g. \
+        {'SiO2': 0.25, 'CaO': 0.25, 'MgO': 0.25, 'FeO': 0.25}
```

Comment:
There is also a way to present write multi-line :param:" descriptions:
:param x: [mole fraction] composition dictionary , e.g.
  {'SiO2': 0.25, 'CaO': 0.25, 'MgO': 0.25, 'FeO': 0.25}
Identing the "{'SiO2': 0.25, 'CaO': 0.25, 'MgO': 0.25, 'FeO': 0.25}" line by 2 spaces will ensure that it is displayed nicely as a 2nd line in 'help()' and sphinx api documentation.